### ISSUE TITLE
Skript Severe Error

### ISSUE THREAD

**Issue by RoVaSte** [SkriptLang/Skript#6049]
### Skript/Server Version

```markdown
[Skript] Server Version: git-Paper-"b4e3b3d" (MC: 1.20.1)
[Skript] Skript Version: 2.7.0 (skriptlang-github)
[Skript] Installed Skript Addons: None
[Skript] Installed dependencies: 
[Skript]  - Vault v1.7.3-b131
[Skript]  - WorldGuard v7.0.9+5934e49
```

### Bug Description

I just tried to reload my skript but the plugin gives me this error. I had 2 addons: SkBee and skript-placeholders, but I am still getting this error after removing them.

### Expected Behavior

My skript should reload properly

### Steps to Reproduce

I am not sure how can I do that. I just edited my code, uploaded to the server and reloaded the skript. 

### Errors or Screenshots

20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! [Skript] Severe Error:
20.09 17:26:37 [Server] [ERROR] #!#! Exception occurred in Skript's main command
20.09 17:26:37 [Server] [ERROR] #!#! Used command: /sk reload rovaste.sk
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Something went horribly wrong with Skript.
20.09 17:26:37 [Server] [ERROR] #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
20.09 17:26:37 [Server] [ERROR] #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
20.09 17:26:37 [Server] [ERROR] #!#! Here is full list of them:
20.09 17:26:37 [Server] [ERROR] #!#! skript-placeholders v1.5.2 (https://github.com/APickledWalrus/skript-placeholders) SkBee v2.18.1 (https://github.com/ShaneBeee/SkBee) 
20.09 17:26:37 [Server] [ERROR] #!#! We could not identify which of those are specially related, so this might also be Skript issue.
20.09 17:26:37 [Server] [ERROR] #!#! You should try disabling those plugins one by one, trying to find which one causes it.
20.09 17:26:37 [Server] [ERROR] #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
20.09 17:26:37 [Server] [ERROR] #!#! In that case, you will be given instruction on how should you report it.
20.09 17:26:37 [Server] [ERROR] #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
20.09 17:26:37 [Server] [ERROR] #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Stack trace:
20.09 17:26:37 [Server] [ERROR] #!#! java.lang.NullPointerException: Cannot invoke "ch.njol.skript.structures.StructVariables$DefaultVariables.getVariables()" because "data" is null
20.09 17:26:37 [Server] [ERROR] #!#!     at Skript.jar//ch.njol.skript.structures.StructVariables.postUnload(StructVariables.java:251)
20.09 17:26:37 [Server] [ERROR] #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.unloadScripts(ScriptLoader.java:845)
20.09 17:26:37 [Server] [ERROR] #!#!     at Skript.jar//ch.njol.skript.ScriptLoader.unloadScript(ScriptLoader.java:864)
20.09 17:26:37 [Server] [ERROR] #!#!     at Skript.jar//ch.njol.skript.SkriptCommand.onCommand(SkriptCommand.java:189)
20.09 17:26:37 [Server] [ERROR] #!#!     at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
20.09 17:26:37 [Server] [ERROR] #!#!     at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155)
20.09 17:26:37 [Server] [ERROR] #!#!     at org.bukkit.craftbukkit.v1_20_R1.CraftServer.dispatchCommand(CraftServer.java:1007)
20.09 17:26:37 [Server] [ERROR] #!#!     at org.bukkit.craftbukkit.v1_20_R1.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64)
20.09 17:26:37 [Server] [ERROR] #!#!     at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:265)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:324)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:308)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2354)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.network.PlayerConnection.lambda$handleChatCommand$21(PlayerConnection.java:2314)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.b(IAsyncTaskHandler.java:59)
20.09 17:26:37 [Server] [ERROR] #!#!     at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.TickTask.run(TickTask.java:18)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1338)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1315)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1308)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.p_(MinecraftServer.java:1286)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1174)
20.09 17:26:37 [Server] [ERROR] #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
20.09 17:26:37 [Server] [ERROR] #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Version Information:
20.09 17:26:37 [Server] [ERROR] #!#!   Skript: 2.7.0 (latest)
20.09 17:26:37 [Server] [ERROR] #!#!     Flavor: skriptlang-github
20.09 17:26:37 [Server] [ERROR] #!#!     Date: 08:55:03.744299
20.09 17:26:37 [Server] [ERROR] #!#!   Bukkit: 1.20.1-R0.1-SNAPSHOT
20.09 17:26:37 [Server] [ERROR] #!#!   Minecraft: 1.20.1
20.09 17:26:37 [Server] [ERROR] #!#!   Java: 17.0.7 (OpenJDK 64-Bit Server VM 17.0.7+7)
20.09 17:26:37 [Server] [ERROR] #!#!   OS: Linux amd64 4.18.0-348.2.1.el8_5.x86_64
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Server platform: Paper
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Current node: null
20.09 17:26:37 [Server] [ERROR] #!#! Current item: null
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Thread: Server thread
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! Language: english
20.09 17:26:37 [Server] [ERROR] #!#! Link parse mode: DISABLED
20.09 17:26:37 [Server] [ERROR] #!#! 
20.09 17:26:37 [Server] [ERROR] #!#! End of Error.
20.09 17:26:37 [Server] [ERROR] #!#! 

### Other

I just removed a list with variables and the skript is now reloading. Why is this happening?

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by sovdeeth** [comment]
You need to provide the error.

**Comment by RoVaSte** [comment]
Sorry, I forgot to do that. I just edited the message.

**Comment by sovdeeth** [comment]
Duplicate of #6013 
Should be fixed in 2.7.1, you might be able to use [the nightly here](https://github.com/SkriptLang/Skript/actions/runs/6250290990) in the meantime
